modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -192,6 +192,12 @@
 			"jsp"
 		]
 	},
+	{
+		"from": "ListUtil.fromArray",
+		"issueKey": "LPS-192403",
+		"newImports": "java.util.ArrayList",
```

Comment:
@MicaelleSilvaa This is wrong, look to the other patterns that have the attribute "newImports". Should be an array, not a string. Your test didn't break because you also forgot to add the new import to the test. Please fix that.